packages/creator/src/helpers/add-content-to-solution.ts

```diff
@@ -455,38 +455,40 @@ export async function _postProcessTaskResource(
     );
   });
 
-  await Promise.all(resourcePromises).then(async (r) => {
-    let resourceString = JSON.stringify(r);
+  await Promise.all(resourcePromises).then(async (resources) => {
+    resources.forEach((r) => {
+      let resourceString = JSON.stringify(r);
 
-    // replace urls first
-    orderedUrls.forEach((url) => {
-      // TypeScript for es2015 doesn't have a definition for `replaceAll`
-      resourceString = (resourceString as any).replaceAll(url, urlVarHash[url]);
-    });
+      // replace urls first
+      orderedUrls.forEach((url) => {
+        // TypeScript for es2015 doesn't have a definition for `replaceAll`
+        resourceString = (resourceString as any).replaceAll(url, urlVarHash[url]);
+      });
 
-    // replace any item ids that aren't already variables
-    Object.keys(itemIds).forEach((k) => {
-      let pattern = new RegExp(`(?<!\\{\\{)${k}`, "g");
-      // TypeScript for es2015 doesn't have a definition for `replaceAll`
-      resourceString = (resourceString as any).replaceAll(pattern, itemIds[k]);
-    });
+      // replace any item ids that aren't already variables
+      Object.keys(itemIds).forEach((k) => {
+        let pattern = new RegExp(`(?<!\\{\\{)${k}`, "g");
+        // TypeScript for es2015 doesn't have a definition for `replaceAll`
```

Comment:
We can remove this comment throughout because we're using es2022 and `replaceAll` came in in es2021